README.md

```diff
@@ -62,10 +62,10 @@ var myApp = angular.module('myModule', ['angular-google-analytics']);
 ### 3- Set your Google Analytics account and start tracking:
 
 ```JavaScript
-myApp.config(function (AnalyticsProvider) {
+myApp.config('AnalyticsProvider', function (AnalyticsProvider) {
    // Add configuration code as desired
    AnalyticsProvider.setAccount('UU-XXXXXXX-X');  //UU-XXXXXXX-X should be your tracking code
-}).run(['Analytics'], function(Analytics) { });
+}).run(['Analytics', function(Analytics) { });
```

Comment:
This needs to be: ```['Analytics', function(Analytics) { }]```

Note the trailing closing square brace after the full function definition.